README.md

```diff
@@ -9,11 +9,12 @@ Each binding will allow you to browse through Git objects of a certain type,
 and select the objects you want to paste to your command-line.
 
 [fzf]: https://github.com/junegunn/fzf
+[fzf-tmux]: https://github.com/junegunn/fzf/blob/master/bin/fzf-tmux
 
 Installation
 ------------
 
-1. Install the latest version of [fzf][fzf]
+1. Install the latest version of [fzf][fzf] and [fzf-tmux][fzf-tmux]
```

Comment:
I see. However, the current sentence can give the wrong impression that the user should install fzf-tmux separately. How about this?

```suggestion
1. Install the latest version of [fzf][fzf] (including [fzf-tmux][fzf-tmux])
```

Reply:
Your revision is more accurate. many thanks!